DefaultTabBar.js

```diff
@@ -45,7 +45,7 @@ const DefaultTabBar = React.createClass({
       accessibilityTraits='button'
       onPress={() => this.props.goToPage(page)}
     >
-      {this.renderTabName(name, page, isTabActive)}
+      {this.props.renderTabName(name, page, isTabActive)}
```

Comment:
I'm pretty sure that someone will decide to override [Button](https://github.com/anyx/react-native-scrollable-tab-view/blob/12a6026848a4edb731e0dc211d923a040757f368/DefaultTabBar.js#L40).
I want to keep only ```renderTab```